designsafe/static/styles/app-card.css

```diff
@@ -0,0 +1,101 @@
+/* STRUCTURE */
+
+/* To load most structure from Core-Styles */
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@1f65993/dist/trumps/s-app-card.css");
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@1f65993/dist/settings/font.css");
+
+/* To support equal-height columns in Bootstrap 3 */
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@1f65993/dist/components/bootstrap.row.css");
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@1f65993/dist/settings/space.css");
```

Comment:
I received an expression of concern about using CDN URLs in general for Core-Styles code in DesignSafe. Use of Core-Styles in DesignSafe is new, and there is preference for copy-paste instead, because it is more reliable and familiar. To adhere to this without loosing a direct paper trail, I will retain the URLs in a comment above the copy-pasted code.

Reply:
Discussion continues in https://github.com/DesignSafe-CI/portal/pull/1168#discussion_r1509384492.